pom.xml

```diff
@@ -355,6 +356,15 @@
             </compilerArgs>
             <showWarnings>true</showWarnings>
             <failOnWarning>false</failOnWarning>
+            <!-- 
+            <annotationProcessorPaths>
+             <path>
+              <groupId>org.immutables</groupId>
+              <artifactId>value</artifactId>
+              <version>${immutables.version}</version>
+              </path>
+            </annotationProcessorPaths>
+            -->
```

Comment:
Can this just be removed?

Reply:
Eggs and chicken:
- needs to be uncomment to work in eclipse. 
- need to be comment for the generator to run and create the builder. 

I would like to keep it because i do not found another way to have it running on the IDE.

Reply:
Not sure what you mean here.  Can you not generate that code with `mvn compile`? This is what I do when using intellij.  I would prefer not to have comment out code without explaining why it's there, but would prefer not to have it at all.